modules/acs-architecture-overview.adoc

```diff
@@ -7,18 +7,21 @@
 
 {product-title} ({product-title-short}) uses a distributed architecture that supports high-scale deployments and is optimized to minimize the impact on the underlying {ocp} or Kubernetes nodes.
 
-.{product-title} architecture for Kubernetes
-image::acs-architecture-kubernetes.png[{product-title} architecture for Kubernetes]
-
 [NOTE]
 ====
 The architecture is slightly different when you install {product-title-short} on Kubernetes and in {ocp}.
 However, the underlying components and the interactions between them remain the same.
 ====
 
-You install {product-title-short} as a set of containers in your {ocp} or Kubernetes cluster. {product-title-short} includes:
+.{product-title-short} architecture for Kubernetes
+
+The following graphic shows the architecture with the StackRox Scanner. For version 4.4, Scanner V4 is available. Installation of Scanner V4 is optional, but provides additional benefits.
+
+image::acs-architecture-kubernetes.png[{product-title} architecture for Kubernetes]
```

Comment:
The updated graphics are not ready yet; the illustrator has pulled them into her next sprint. Will update doc when they are received.